include/crm/common/xml_internal.h

```diff
@@ -354,7 +354,7 @@ pcmk__xe_next(const xmlNode *child)
 xmlNode *pcmk__xe_create(xmlNode *parent, const char *name);
 xmlNode *pcmk__xml_copy(xmlNode *parent, xmlNode *src);
 
-void pcmk__xe_set_content(xmlNode *node, const char *content);
+void pcmk__xe_set_content(xmlNode *node, const char *format, ...);
```

Comment:
add a G_GNUC_PRINTF annotation

Reply:
I hope it's acceptable to pass an empty argument list with this. The second argument of `G_GNUC_PRINTF` is the index of the first argument to check against the format string. The gcc doc doesn't explicitly say that it's okay to pass a non-zero `first-to-check` and then have no arguments to check: https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html#index-format-function-attribute

I will assume it's fine since `printf()` allows for example `printf("hello world\n");`